webapp/src/DataTransferObject/ContestProblemPut.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+namespace App\DataTransferObject;
+
+use OpenApi\Attributes as OA;
+
+#[OA\Schema(required: ['label'])]
+class ContestProblemPut
+{
+    public function __construct(
+        #[OA\Property(description: 'The label of the problem to add to the contest')]
+        public readonly string $label,
+        #[OA\Property(description: 'Human readable color of the problem to add. Will be overwritten by `rgb` if supplied', nullable: true)]
+        public readonly ?string $color,
+        #[OA\Property(description: 'Hexadecimal RGB value of the color of the problem to add. Will be used if both `color` and `rgb` are supplied', nullable: true)]
```

Comment:
```suggestion
        #[OA\Property(description: 'Hexadecimal RGB value of the color of the problem to add. Overrules `color` if supplied', nullable: true)]
```

Has nothing to do with your PR.